src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
 
+import Calculator from './components/Calculator';
+
 function App() {
```

Comment:
- [ ] This function should be class-based. The same way you did with the Calculator component. Please fix it.